src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/VisitSchedulerExceptionHandler.kt

```diff
@@ -262,3 +262,18 @@ data class ErrorResponse(
   ) :
     this(status.value(), errorCode, userMessage, developerMessage)
 }
+
+data class ValidationErrorResponse(
+  override val status: Int,
+  override val errorCode: Int? = null,
+  override val userMessage: String? = null,
+  val developerMessages: List<String>,
```

Comment:
should this field be renamed to say moreInfo?

Reply:
see comment below